lib/Resource.js

```diff
@@ -4,6 +4,6 @@ module.exports = Resource;
 
 function Resource(id, data, expires) {
     this.$id = id;
-    this.$expires = parseInt(expires, 10) || 0;
+    this.$expires = (typeof expires === 'number' ? expires : undefined);
```

Comment:
`isNaN` only returns `true` for `NaN` and `undefined`. If it returns `false`, someone may have passed in `null` and this would result in an invalid `$expires` value. Checking just for numbers is safer.